features/admin.applications.v1/components/settings/sign-on-methods/sign-on-methods.tsx

```diff
@@ -110,10 +110,3 @@ export const SignOnMethods: FunctionComponent<SignOnMethodsPropsInterface> = (
         />
     );
 };
-
-/**
- * Default props for the application sign-on-methods component.
- */
-SignOnMethods.defaultProps = {
```

Comment:
Verify if removing the `componentid` cause any test suite failures.

Reply:
Added default `componentId`s to `sign-on-methods-wrapper.tsx` and `sign-on-methods.tsx`. 

The `componentId` passed from `edit-application.tsx` is forwarded to `sign-on-methods-core.tsx`  without any modifications. This ensures that the test suite failures are not caused due to changes in the `componentId`.